qa/workunits/objectstore/test_fuse.sh

```diff
@@ -43,46 +43,46 @@ test -d $MNT/meta/all
 test -d $MNT/meta/by_bitwise_hash
 
 # create object
-mkdir $MNT/meta/all/@-1:7b3f43c4:::osd_superblock:0@
-test -e $MNT/meta/all/@-1:7b3f43c4:::osd_superblock:0@/data
-test -d $MNT/meta/all/@-1:7b3f43c4:::osd_superblock:0@/attr
-test -d $MNT/meta/all/@-1:7b3f43c4:::osd_superblock:0@/omap
-test -e $MNT/meta/all/@-1:7b3f43c4:::osd_superblock:0@/bitwise_hash
-test -e $MNT/meta/all/@-1:7b3f43c4:::osd_superblock:0@/omap_header
+mkdir $MNT/meta/all/#-1:7b3f43c4:::osd_superblock:0#
```

Comment:
just a note for future readers, we started to use `#` since 4bd31faee0b0de908069287fa14c74dc889b41e5.